contrib/rrdtool/template.py

```diff
@@ -0,0 +1,30 @@
+pkgname = "rrdtool"
+pkgver = "1.8.0"
+pkgrel = 0
+build_style = "gnu_configure"
+make_dir = "."
+hostmakedepends = [
+    "automake",
+    "bash",
+    "gettext-devel",
+    "groff",
+    "libtool",
+    "pkgconf",
+]
+makedepends = ["glib-devel", "libxml2-devel", "pango-devel", "python-devel"]
+pkgdesc = "Round Robin Database"
+maintainer = "yanchan09 <yan@omg.lol>"
+license = "custom:GPL-2.0-or-later-with-FLOSS-Exception"
```

Comment:
bit unsure about this, maybe it'd be fine to just specify `GPL-2.0-or-later` or something along the lines of `GPL-2.0-or-later AND custom:FLOSS-Exception`

Reply:
for reference, the license text is at https://github.com/oetiker/rrdtool-1.x/blob/master/COPYRIGHT

Reply:
yes, use that instead, like in console-setup

or WITH like in sane-backends which might be invalid but who cares